user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -609,7 +609,8 @@ def __build_global_report_summary(self,
 
         if not apps_working_set.empty:
             self.logger.info('Generating GPU Estimated Speedup and Savings as %s', csv_out)
-            apps_working_set.to_csv(csv_out)
+            # we can use the general format as well but this will transform numbers to E+. So, stick with %f
+            apps_working_set.to_csv(csv_out, float_format='%.3f')
```

Comment:
question: why did you choose `.3f` instead of `.2f` as normally we present 2 decimals in output?

Reply:
I thought the CSV would have 3 to make it more detailed compared to the STDOUT.
I changed it to 2f in the last commit.